src/org/labkey/test/tests/InlineImagesAssayTest.java

```diff
@@ -172,14 +173,22 @@ public final void testAssayInlineImages() throws Exception
         clickButton("Submit");
         waitForElement(DataRegionTable.updateLinkLocator()); // Wait to make sure the grid has been rendered.
 
+        // add columns to the grid to get assay and run id for file verification path
+        DataRegionTable list = new DataRegionTable("Data", getDriver());
+        CustomizeView customizeView = list.openCustomizeGrid();
+        customizeView.showHiddenItems();
+        customizeView.addColumn(new String[]{"Run", "RowId"});
+        customizeView.addColumn(new String[]{"Run", "Protocol", "RowId"});
+        customizeView.applyCustomView();
+        String helpJpgFilePath = "AssayId_" + list.getDataAsText(0, "Run/Protocol/RowId") + "/RunId_" + list.getDataAsText(0, "Run/RowId") + "/" + HELP_JPG_FILE.getName();
```

Comment:
Is the file path OS dependent? I'm not sure that it is but want to check.

In some LKSM test failures where the OS path was an issue I addressed like this:
```
right = SystemUtils.IS_OS_WINDOWS ?
                            "sampletype\\" + right :
                            "sampletype/" + right; 
```

Reply:
This code looks to have passed on both PG and SQLServer on TC so I think we are good.

Reply:
Good call, Dan. Chris tested this locally on his Windows machine and it failed. He pushed a change to us the OS file separator instead and TC looks happy with that.